ntoskrnl/CMakeLists.txt

```diff
@@ -3,6 +3,11 @@ PROJECT(NTOS)
 
 include(ntos.cmake)
 
+if (CMAKE_C_COMPILER_ID MATCHES "GNU|Clang")
```

Comment:
This seems to be an interesting way (for our other cmakelists) to match for either GCC or Clang...

Reply:
This is how we should do it 🙂